.editorconfig rule ktlint_standard_multiline-expression-wrapping break ktlintCheck

Try to migrate to ktlint 1.0.0.

`./gradlew ktlintCheck` failed when I add rule into `.editorconfig` file:

```
ktlint_standard_multiline-expression-wrapping = disabled
```

whole .editorconfig file looks like this:
```
root = true

[*]
insert_final_newline = true
indent_style = space
trim_trailing_whitespace = true

# ! Don't put space between [kt,kts] it will break file definition!
[*.{kt,kts}]
max_line_length = 130
indent_size = 4
continuation_indent_size = 4
wildcard_import_limit = 0
# don't use wildcard for imports
ij_kotlin_name_count_to_use_star_import = 999
ij_kotlin_name_count_to_use_star_import_for_members = 999
ij_kotlin_allow_trailing_comma = true
# ij_kotlin_imports_layout = *
ktlint_code_style = ktlint_official
ktlint_experimental = disabled # Enable/disable all `experimental` rules from all rule sets provided by KtLint or other rule providers
ktlint_standard_import-ordering = disabled
ktlint_standard_blank-line-before-declaration = disabled
ktlint_standard_filename = disabled
ktlint_standard_final-newline = disabled
ktlint_standard_max-line-length = disabled
ktlint_standard_function-signature = disabled
# https://pinterest.github.io/ktlint/1.0.0/rules/standard/#multiline-expression-wrapping I really hate that rule :)
ktlint_standard_multiline-expression-wrapping = disabled
ktlint_standard_no-empty-first-line-in-class-body = disabled
ktlint_standard_trailing-comma-on-call-site = disabled

[*.kts]
max_line_length = 200

[*.xml]
indent_size = 4
continuation_indent_size = 4
```

error from stacktrace:
```
Caused by: org.gradle.api.GradleException: KtLint failed to parse file: D:\Projects\digilab\hoppygo\hoppygo\repository\build.gradle.kts
        at org.jlleitschuh.gradle.ktlint.worker.KtLintWorkAction.execute(KtLintWorkAction.kt:126)
        at org.gradle.workers.internal.DefaultWorkerServer.execute(DefaultWorkerServer.java:63)
        at org.gradle.workers.internal.AbstractClassLoaderWorker$1.create(AbstractClassLoaderWorker.java:54)
        at org.gradle.workers.internal.AbstractClassLoaderWorker$1.create(AbstractClassLoaderWorker.java:48)
        at org.gradle.internal.classloader.ClassLoaderUtils.executeInClassloader(ClassLoaderUtils.java:100)
        at org.gradle.workers.internal.AbstractClassLoaderWorker.executeInClassLoader(AbstractClassLoaderWorker.java:48)
        at org.gradle.workers.internal.IsolatedClassloaderWorker.run(IsolatedClassloaderWorker.java:49)
        at org.gradle.workers.internal.IsolatedClassloaderWorker.run(IsolatedClassloaderWorker.java:30)
        at org.gradle.workers.internal.WorkerDaemonServer.run(WorkerDaemonServer.java:96)
        at org.gradle.workers.internal.WorkerDaemonServer.run(WorkerDaemonServer.java:65)
        at org.gradle.process.internal.worker.request.WorkerAction$1.call(WorkerAction.java:138)
        at org.gradle.process.internal.worker.child.WorkerLogEventListener.withWorkerLoggingProtocol(WorkerLogEventListener.java:41)
        at org.gradle.process.internal.worker.request.WorkerAction.lambda$run$0(WorkerAction.java:135)
        at org.gradle.internal.operations.CurrentBuildOperationRef.with(CurrentBuildOperationRef.java:80)
        at org.gradle.process.internal.worker.request.WorkerAction.run(WorkerAction.java:127)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
        at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
        at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:182)
        at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:164)
        at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:414)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
        at org.gradle.internal.concurrent.AbstractManagedExecutor$1.run(AbstractManagedExecutor.java:47)
        ... 3 more
Caused by: java.lang.IllegalStateException: Skipping rule(s) which are depending on a rule which is not loaded. Please check if you need to add additional rule sets before creating an issue.
  - Rule with id 'RuleId(value=standard:string-template-indent)' requires rule with id 'RuleId(value=standard:multiline-expression-wrapping)' to be loaded
        at com.pinterest.ktlint.rule.engine.internal.rulefilter.RunAfterRuleFilter.filter(RunAfterRuleFilter.kt:77)
        at com.pinterest.ktlint.rule.engine.internal.rulefilter.RuleFilterKt.applyRuleFilters(RuleFilter.kt:15)
        at com.pinterest.ktlint.rule.engine.internal.RuleExecutionContext$Companion.createRuleExecutionContext$ktlint_rule_engine(RuleExecutionContext.kt:192)
        at com.pinterest.ktlint.rule.engine.api.KtLintRuleEngine.lint(KtLintRuleEngine.kt:94)
        at org.jlleitschuh.gradle.ktlint.worker.KtLintInvocation100.invokeLint(KtLintInvocation100.kt:47)
        at org.jlleitschuh.gradle.ktlint.worker.KtLintWorkAction.execute(KtLintWorkAction.kt:122)
        ... 28 more
```

If I've added `ktlint_standard_string-template-indent = disabled` into EditorConfig too, it start working again, but this error is shown only in stacktrace, so without stracktrace you have no idea what's wrng and why `ktlintCheck` failed.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 ## [Unreleased]
 
 -   update latest version text file manually [#709](https://github.com/JLLeitschuh/ktlint-gradle/pull/709)
+-   Improve error logging [#711](https://github.com/JLLeitschuh/ktlint-gradle/pull/711)
 
 ## [11.6.0] - 2023-09-18
 

--- a/plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/worker/KtLintWorkAction.kt
+++ b/plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/worker/KtLintWorkAction.kt
@@ -123,6 +123,7 @@ abstract class KtLintWorkAction : WorkAction<KtLintWorkAction.KtLintWorkParamete
                     results.add(result)
                 }
             } catch (e: RuntimeException) {
+                logger.error(e.message)
                 throw GradleException(
                     "KtLint failed to parse file: ${it.absolutePath}",
                     e

